<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EtoileVideoPlayer</class>
 <widget class="QWidget" name="EtoileVideoPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>499</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>EtoileVideoPlayer</string>
  </property>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>501</width>
     <height>121</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="player">
    <attribute name="title">
     <string>Tab 1</string>
    </attribute>
    <widget class="QPushButton" name="stop">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>stop</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>open</string>
     </property>
    </widget>
    <widget class="Phonon::SeekSlider" name="seekSlider">
     <property name="geometry">
      <rect>
       <x>23</x>
       <y>60</y>
       <width>341</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLCDNumber" name="playtime">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>50</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="play">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>75</width>
       <height>23</height>
      </rect>
     </property>
     <property name="text">
      <string>play</string>
     </property>
    </widget>
    <widget class="Phonon::VolumeSlider" name="volumeSlider">
     <property name="geometry">
      <rect>
       <x>360</x>
       <y>20</y>
       <width>109</width>
       <height>22</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="PlayListWidget" name="playlist">
    <attribute name="title">
     <string>Tab 2</string>
    </attribute>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Phonon::SeekSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/seekslider.h</header>
  </customwidget>
  <customwidget>
   <class>Phonon::VolumeSlider</class>
   <extends>QWidget</extends>
   <header location="global">phonon/volumeslider.h</header>
  </customwidget>
  <customwidget>
   <class>PlayListWidget</class>
   <extends>QWidget</extends>
   <header>PlayListWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
